mmv1/products/certificatemanager/api.yaml

```diff
@@ -67,9 +67,23 @@ objects:
         name: 'description'
         description: |
           A human-readable description of the resource.
+      - !ruby/object:Api::Type::String
+        name: 'createTime'
+        output: true
+        description: |
+          Creation timestamp of a DNS Authorization. Timestamp in RFC3339 UTC "Zulu" format,
+          with nanosecond resolution and up to nine fractional digits.
+          Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+      - !ruby/object:Api::Type::Time
+        name: 'updateTime'
+        description: |
+          Update timestamp of a DNS Authorization. Timestamp in RFC3339 UTC "Zulu" format,
+          with nanosecond resolution and up to nine fractional digits.
+          Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
+        output: true
```

Comment:
Are these two needed for any particular reason? Like are there any other resources that depend on these?

Reply:
delete updateTime and createTime in all reasources